{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -143,6 +144,13 @@ enter_docker:
 test_environment:
 	$(PYTHON_INTERPRETER) test_environment.py
 
+## Render and insert report notebooks into DSDocs Jekyll site.
+insert_into_dsdocs:
+	$(PYTHON_INTERPRETER) "${DSDOCS_DIR}/scripts/render_and_insert_all.py" \
+	    ./notebooks/report/ \
```

Comment:
I believe one of our old JIRA tickets (or at least, @davidzornek has mentioned it before) includes removing the notebooks/report and notebooks/exploratory subdirectories and just having notebooks/. If that is still what we want to do, should we simply remove those subdirectories as part of this PR and refer to notebooks/ only?

Reply:
I think so.

Reply:
Perhaps so. :)